labrad/types/__init__.py

```diff
@@ -0,0 +1 @@
+from labrad.types.types import *
```

Comment:
Rather than importing `*`, I'd prefer to explicitly list the imported names here, even if the list is longish.